data/json/overmap_specials.json

```diff
@@ -934,7 +934,7 @@
         "rotate" : true, "//":"allow rotation",
         "unique" : false, "//":"only allow one per city",
         "required" : false, "//":"is this special required for valid city sizes",
-        "flags" : [ "ROTATE", "UNIQUE", "CLASSIC", "REQUIRED" ]
+        "flags" : ["CLASSIC"]
```

Comment:
Why boolean ```"unique" : false``` but flag "UNIQUE" exists? (and the same for is applicable to ```required``` var).

What is desired conditions here?